qtlib/about_box.py

```diff
@@ -69,6 +69,21 @@ def _setupUi(self):
         self.verticalLayout.addWidget(self.label_3)
         self.label_3.setText(tr("Licensed under GPLv3"))
         self.label = QLabel(self)
+        self.label_4 = QLabel(self)
+        self.label_4.setWordWrap(True)
+        self.label_4.setTextFormat(Qt.RichText)
+        self.label_4.setOpenExternalLinks(True)
+        self.label_4.setText(tr(
```

Comment:
This is the first time something like this has been done in this project, most other non-code contributes have been in CREDITS.  Granted the GPL ones are a bit different than that intention.  Not 100% sure on doing it this way.

Reply:
I had overlooked the CREDITS file and I was so invested in displaying them there. Will remove.

Reply:
However, maybe it would be nice to display the whole credits in that dialog box later?

Reply:
Yeah I don't think I am necessarily opposed to that, although I think it needs to be thought of how to do it in a way that would be "fair".   Since those making code contributes are not in credits only the non-code items.  I think if we were going to do something like that it probably should include all the contributions.  I could see that being something to add post 4.0.5.

Reply:
Right, I will have to gather all contributors to the git repo (not hat difficult). I could make a tab for code contributors and maintainers, another tab for translations, another tab for everything else. Will keep a note about this in my TODO list for later.